doc/xanadu_theme/static/xanadu.css_t

```diff
@@ -1,7 +1,7 @@
 /* Sphinx themes
 -------------------------------------------------- */
 @font-face {
-  font-family: 'Product Sans';
+  font-family: 'Noto Sans';
```

Comment:
The current font doesn't look like Product Sans at all; where is it being used?

Reply:
Hey @speller26, this was a bug that we have fixed and updated across all the CSS. This change simply brings the Braket docs inline with the rest of the PL documentation